gateway/crates/gateway-binary/src/main.rs

```diff
@@ -212,10 +213,6 @@ fn otel_layer_reload<S>(
         grafbase_telemetry::otel::opentelemetry::global::set_meter_provider(meter_provider);
         grafbase_telemetry::otel::opentelemetry::global::set_tracer_provider(tracer.provider.clone());
 
-        if let Some(logger) = logger {
-            grafbase_telemetry::otel::opentelemetry::global::set_logger_provider(logger.provider.clone());
```

Comment:
No longer needed?

Reply:
that's suspicious, why is logging removed?

Reply:
Yeah they removed this from the API completely. Seems it works without.